sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DocumentCollection.java

```diff
@@ -427,12 +430,18 @@ public DocumentCollection getWrappedItem() {
 
         private void writeObject(ObjectOutputStream objectOutputStream) throws IOException {
             documentCollection.populatePropertyBag();
-            objectOutputStream.writeObject(documentCollection.getPropertyBag());
+            ObjectNode docCollectionNode = OBJECT_MAPPER.createObjectNode();
+            docCollectionNode.set("col", documentCollection.getPropertyBag());
+            docCollectionNode.set("altLink", TextNode.valueOf(documentCollection.getAltLink()));
```

Comment:
perhaps we should create constant for thse.

Reply:
ACK - will combinein the next PR